integration-tests/qm/package.json

```diff
@@ -18,7 +18,7 @@
     "prisma": "4.0.0",
     "react": "18.0.0",
     "react-dom": "18.0.0",
-    "react-query": "3.39.0"
+    "react-query": "4.0.0-beta.23"
```

Comment:
Shouldn't this be changed to @tanstack/react-query ?

Reply:
Yeah, this was with the last react-query version (before official 4.0.0)